core/dungeon-masters-guide/items/items-wondrous.xml

```diff
@@ -4085,4 +4085,323 @@
+			<p class="indent">Benefits granted by the <em>Book of Exalted Deeds</em> last only as long as you strive to do good. If you fail to perform at least one act of kindness or generosity within the span of 10 days, or if you willingly perform an evil act, you lose all the benefits granted by the book.</p>
+			<p><b>Random Properties.</b> The <em>Book of Exalted Deeds</em> has the following random properties:</p>
+			<ul>
+				<li>2 minor beneficial properties</li>
+				<li>2 major beneficial properties</li>
+			</ul>
+			<p class="indent"><i><b>Increased Wisdom.</b></i> After you spend the requisite amount of time reading and studying the book, your Wisdom score increases by 2, to a maximum of 24. You can't gain this benefit from the book more than once.</p>
+			<p class="indent"><i><b>Enlightened Magic.</b></i> Once you've read and studied the book, any spell slot you expend to cast a cleric or paladin spell counts as a spell slot of one level higher.</p>
+			<p class="indent"><i><b>Halo.</b></i> Once you've read and studied the book, you gain a protective halo. This halo sheds bright light in a 10-foot radius and dim light for an additional 10 feet. You can dismiss or manifest the halo as a bonus action. While present, the halo gives you advantage on Charisma (Persuasion) checks made to interact with good creatures and Charisma (Intimidation) checks made to interact with evil creatures. In addition, fiends and undead within the halo's bright light make attack rolls against you with disadvantage.</p>
+			<p class="indent"><i><b>Destroying the Book.</b></i> It is rumored that the <em>Book of Exalted Deeds</em> can't be destroyed as long as good exists in the multiverse. However, drowning the book in the River Styx removes all writing and imagery from its pages and renders the book powerless for 1d100 years.</p>
+			<div class="reference">
+				<div element="ID_WOTC_DMG_ARTIFACT_TABLE" />
+			</div>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="5">5 lb.</set>
+			<set name="type">Wondrous Items</set>
+			<set name="rarity">Artifact</set>
+			<set name="attunement" addition="by a creature of good alignment">true</set>
+		</setters>
+	</element>
+
+	<element name="Book of Vile Darkness" type="Magic Item" source="Dungeon Master’s Guide" id="ID_WOTC_DMG_MAGIC_ITEM_BOOK_OF_VILE_DARKNESS">
```

Comment:
Missing rules for the ability score effects

Reply:
As there are technically 6 different variants of the book do we want 6 different books or do we want to make a select for each of the 6 variants and if it's the select where should the select go (Ie. Class/Race)

Reply:
I would say lets do a select under the Ability Score section

Reply:
I've just found a comment from nick when I was first making these files saying not to do the ASI from these

Reply:
https://github.com/AuroraLegacy/elements/pull/7#discussion_r526394463